File: examples/ts/fetch-orderbook.ts
import ccxt from '../../ts/ccxt.js';

// AUTO-TRANSPILE //

async function example () {
    const exchange = new ccxt.binance ({});
    const ob = await exchange.fetchOrderBook ('BTC/USDT', 3);
    const asks = ob['asks'];
    const bids = ob['bids'];
    console.log (asks);
    console.log (bids);
}

example ();

File: examples/c#/FetchOrderBook.cs
using ccxt;
using Newtonsoft.Json;

namespace examples;

partial class Examples
{
    public static void FetchOrderBook()
    {
        var exchange = new Binance();
        var ob = exchange.FetchOrderBook("BTC/USDT");
        ob.Wait();
        Console.WriteLine(JsonConvert.SerializeObject(ob.Result, Formatting.Indented));
    }
}